app/containers/FontSizeScale.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { ipcRenderer } from 'electron';
+
+class FontSizeScale extends React.Component {
+  static STEP = 0.1;
```

Comment:
Offhand, I'd just do these as `const` and not class properties. Do we need them to be attached to the class?